### PR TITLE
Milestone for complete project adding Bootstrap

### PR DESCRIPTION
**Pull request by harshdeepkanhai**
Used Bootstrap to build the Newsweek clone

### CODE REVIEW

File: assets/stylesheets/main.css [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+.col-item,
+.mag-start,
+.wrap {
+  position: relative;
+}
+
+/*
+.content-auto {
+    content-visibility: auto;
+} */
```
Comment (kevcap) [inline assets/stylesheets/main.css]: - [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

#### Hey team,  your project is looking really good, however, to accomplish as the actual page, you required to do these changes, listed below:
 
_Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, to get more knowledge and have a better project to add at your portfolio.
 
 
## [REQUIRED CHANGES]
- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */ 
image


- [ ] Kindly remember to remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
Take a look at this example:
image
Your HTML will be something like this: 
image
Apply the same logic above in **ALL** those repeated rules listed below:
image


#### General issues:
- [x] In Large, medium and small screens, your page is missing the EDITOR'S PICK and those other sections such as U.S., World, Business and so on (check that gif made at medium screens)
##### Yours
ezgif com-optimize


##### Original one:
ezgif com-optimize(1)

#### Large screen issues:
- [x] At the bottom of the page, the **newsletter image is too shrinked** take care of that! ⤵ 😎
##### Yours
image
##### Original
image

#### Medium screens
- [x] Please Fix that overlapping ⤵
image
- [x] At the bottom of the page, the **newsletter image is too shrinked** take care of that! ⤵ 😎
image
- [x] At the "The debate" and "Opinion" section, those avatar images are too big, fix that.
##### Yours
image
##### Original one
image

#### Small screen issues:
- [x] The first section should be Top Story, not Featured Stories ⤵
##### Your's:
image
##### Original one:
image

- [x] In small screens your your footer don't look like the original one, because the items are too spaced, kindly improve it
##### Yours:
image
##### Original one:
image
- [x] Your page is facing **overflow at the horizontal axis**,and the **newsletter image is too shrinked** take care of that! ⤵ 😎

 [Image from Gyazo](https://gyazo.com/5b569feeaac31ae8ce80112fcb6f1c7a)
`Kev tip: For fix overflow issues, use this rule: * { border: 1px solid red }, so you'll be able to see the limits of all your elements and spot those who are overflowing!`


## [OPTIONAL CHANGES]
- [ ] Improve the layout (e.g.: sizings, spacing, font, icons) to make it more similar to the original page;
- [ ] Add a favicon to make it more similar to the original page;

## Please make the required changes and resubmit for a new code review once you are ready💪



## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

File: assets/stylesheets/main.css [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+  margin-right: 15px;
+}
+
+.headshot img {
+  width: 100%;
+}
+
+/* div {
+      font: 17px/21px "Roboto Condensed", sans serif;
+  } */
```
Comment (kevcap) [inline assets/stylesheets/main.css]: - [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
# Status: Changes requested :red_circle: 

### Hello @harshdeepkanhai :wave: 

### Thank you for making almost all of the changes requested by the previous TSE, but we still have some details to change. Please read my suggestions below.

## All screens

- [x] You still have a horizontal scroll, please fix it. Think about what could be causing it, maybe it's a container that's causing that?

- [x] You fixed only one of these images in the 'The Debate' section, kindly fix the others:

Screenshot from 2020-09-03 14-57-48
<br>

## General

- [ ] Please fix your screenshot in your README file, it's not showing.

- [ ] Please fix your Live Link, it's not currently working.

### Kindly make the requested changes and submit for another review, please :pray: 

### I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 
## Great job so far :clap: :clap: :+1: but you need to work more on your project to make it looks perfect :100:.

- [x] on mobile view the editor's pick section is not taking the full width of the page, make it like that.

| *Clone* | *Original* |
|:---:|:---:|
| image | image |

- [x] on mobile view also, you need to give more with to the newsletter form and also give it some border:

| *Clone* | *Original* |
|:---:|:---:|
| image | image |

- [x] on the tablet view, the debate section should take the full width of the page:

| *Clone* | *Original* |
|:---:|:---:|
| image | image |

- [x] on the tablet view also, the editor's pick section should take only two articles in a row, not four.

| *Clone* | *Original* |
|:---:|:---:|
| image | image |

- [x] also on the tablet view, you need to fix the newsletter and subscribe sections.

| *Clone* | *Original* |
|:---:|:---:|
| image | image |

### Please make all the required changes before you submit another code review.
## Happy Coding :tophat:

**Review by MuriloRoque — APPROVED** [review]
# Status: Approved :green_circle: 

### Hello again @harshdeepkanhai :wave: 

### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: **Just please don't forget to update the Live Link in your README file** :+1: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**